src/modules/drawing/DrawingModule.vue

```diff
@@ -418,6 +444,11 @@ export default {
                 }
             }
         },
+        async openDrawingOnAdminId(layer) {
+            if (layer?.adminId && !this.isDrawingOpen && !layer?.isLegacy()) {
+                await this.setShowDrawingOverlay(true)
+            }
+        },
```

Comment:
There are a few times where this triggers incorrectly the drawing menu:
- When changing the language after that a drawing has been drawn. I think it is because internally the layers are removed and readded when changing the language.
- If you have multiple drawings and move them around in the active layers list, it will automatically open the drawing mode as soon as a new drawing with admin ID is at the top of the stack, which is unwanted.

Ideally the drawing mode should only be triggered someone pastes an URL with an adminid in the url bar.

Reply:
Good findings I'll try to find a solutions for those use cases.

Reply:
@jedef this should be fixed.